.expeditor/config.yml

```diff
@@ -22,3 +25,7 @@ merge_actions:
   - built_in:build_gem:
       only_if:
         - built_in:bump_version
+
+promote:
+  action:
```

Comment:
Shouldn't this be:

```yaml
artifact_actions:
  promoted_to_stable:
    - built_in:publish_rubygems
```

That matches what we have in various other projects:
https://github.com/chef/chef/blob/master/.expeditor/config.yml#L71-L73
https://github.com/chef/chef-dk/blob/master/.expeditor/config.yml#L66-L68
https://github.com/chef/inspec/blob/master/.expeditor/config.yml#L59-L60

Reply:
I talked to @tduffield about this - since we have no artifacts (rpms, etc) there is no 'artifact_action' for this, thus we need the `promote` primitive.

Reply:
Sounds good.